assets/js-translations/teacher.fi.json

```diff
@@ -54,7 +54,7 @@
   "Organisaatio",
 
   "Student ID":
-  "Opiskelijanumero",
+  "Opiskelija\u00ADnumero",
```

Comment:
"Opiskelijanumero" is without hyphen

Reply:
`\u00AD` is a soft hyphen or a syllable hyphen, so if the word fits on a single line, it's not there, but if it doesn't it appears with the word break. (Basically, this is a way to get the cell to be less wide and still follow grammar rules.)

Reply:
ok, then everything ok.